optuna/integration/mlflow.py

```diff
@@ -1,12 +1,14 @@
+from typing import Optional
+
 import optuna
 from optuna._experimental import experimental
 from optuna._imports import try_import
-from optuna import structs
+from optuna._study_direction import StudyDirection
```

Comment:
How about importing from `optuna.study.StudyDirection`?

Reply:
Hmm, maybe just I'm stupid, but it doesn't sound good by staring at this.

https://github.com/optuna/optuna/blob/master/optuna/study.py#L15

Reply:
Ah, the message recommends that. Gotcha.